src/app/pages/about/about.component.html

```diff
@@ -12,9 +12,9 @@ <h1 class="mb-3" id="title">About Us</h1>
         pandemic leading to lockdowns and social distancing, we wanted to create an alternative way for graduating
         students (and
         others!) to be able to send each other messages in a medium that they can cherish with them as they continue on
-        in their journeys.
+        in their journeys. <br><br>
+        Want to get in touch with us? Visit the <a routerLink="/contact">Contact Us</a> page.
       </p>
-      <p>Want to get in touch with us? Visit the <a routerLink="/contact">Contact Us</a> page.</p>
     </div>
   </div>
 </div>
```

Comment:
Add padding to the bottom to separate content from the footer a bit more.
I'm not sure why Github isn't letting me add a suggestion to this particular line (line 23)

Change
`<div class="mb-3" id="meet-the-team">`

to
`<div class="mb-5" id="meet-the-team">`